app/src/main/java/com/example/parkingsystem/fragment/ParkingSizeDialogFragment.kt

```diff
@@ -47,11 +46,11 @@ class ParkingSizeDialogFragment : DialogFragment() {
         private const val LISTENER_KEY = "LISTENER_KEY"
 
         fun newInstance(listener: ParkingDialogListener): ParkingSizeDialogFragment {
-            val dialog = ParkingSizeDialogFragment()
-            val bundle = Bundle()
-            bundle.putSerializable(LISTENER_KEY, listener)
-            dialog.arguments = bundle
-            return dialog
+            return ParkingSizeDialogFragment().apply {
+                val bundle = Bundle()
+                bundle.putSerializable(LISTENER_KEY, listener)
+                this.arguments = bundle
+            }
         }
```

Comment:
instead of return here, you can use "="
also, you can reduce this again using apply for Bundle()

Reply:
Done